osu.Game.Tests/Visual/Navigation/TestSceneButtonSystemNavigation.cs

```diff
@@ -16,6 +18,103 @@ public partial class TestSceneButtonSystemNavigation : OsuGameTestScene
     {
         private ButtonSystem buttons => ((MainMenu)Game.ScreenStack.CurrentScreen).ChildrenOfType<ButtonSystem>().Single();
 
+        private GlobalActionContainer globalActionContainer => Game.ChildrenOfType<GlobalActionContainer>().First();
+
+        [Test]
+        public void TestKeyboardNavigation()
+        {
+            AddAssert("state is initial", () => buttons.State == ButtonSystemState.Initial);
```

Comment:
We prefer to write assertions in nunit matcher style these days.

```suggestion
            AddAssert("state is initial", () => buttons.State, () => Is.EqualTo(ButtonSystemState.Initial));
```

Brings better error messaging and all that. I'm not commenting this everywhere, hopefully you get the idea.